backend/api/v1/endpoints/connections.py

```
@@ -35,14 +35,24 @@ def reject_connection(request_id: int, db: Session = Depends(get_db), current_us
+def list_connections(db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
+    """Fetch user connections."""
+    try:
+        return get_connections(db, current_user.id)  # ✅ Use `current_user.id`
+    except Exception as e:
+        logging.error(f"Error fetching connections: {str(e)}")
+        raise HTTPException(status_code=500, detail="Internal Server Error")
+
+
 @router.get("/users")
-def get_users(current_user: User = Depends(get_current_user), db: Session = Depends(get_db)):
+def get_users(db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_users`

(ANN201)